tests/unit/cluster/slot-stats.tcl

```diff
@@ -511,6 +839,10 @@ start_cluster 1 0 {tags {external:skip cluster}} {
         # When cluster-slot-stats-enabled config is disabled, you cannot sort using advanced metrics.
         set orderby "cpu-usec"
         assert_error "ERR*" {R 0 CLUSTER SLOT-STATS ORDERBY $orderby}
+        set orderby "network-bytes-in"
+        assert_error "ERR*" {R 0 CLUSTER SLOT-STATS ORDERBY $orderby}
+        set orderby "network-bytes-out"
```

Comment:
I think we're missing tests here to make sure we can order by network bytes in/out